src/hu_nmt/data_augmentator/graph_mappers/edge_mapper.py

```diff
@@ -0,0 +1,161 @@
+import networkx as nx
+from collections import defaultdict, Counter
+import nltk
+from typing import List
+
+from hu_nmt.data_augmentator.graph_mappers.graph_similarity_base import GraphSimilarityBase, Edge, Node
+
+
+class EdgeMapper(GraphSimilarityBase):
+    def __init__(self):
+        self.dep_weights = defaultdict(lambda: 1)
+
+        self.dep_weights['nsubj'] = 3
+        self.dep_weights['obj'] = 3
+        self.dep_weights['nmod'] = 2
+        self.dep_weights['obl'] = 2
+        self.dep_weights['case'] = 2
+        self.dep_weights['amod'] = 2
+        self.dep_weights['advmod'] = 2
```

Comment:
can we document how these were calculated and add comments here? Maybe a short explanation and a link pointing to a notebook with the analysis is fine.

Reply:
The first two are not relevant anymore as we compare subgraphs of the nsubj and obj edge. Kornai suggested the rest as these are the important edge types.

Reply:
I see. Can we still put this as a comment? That we assume these are the important dependency relations in the subtrees of an nsubj and an obj? And we can remove the unused weights.